doc/classes/Node.xml

```diff
@@ -182,6 +182,32 @@
 				[b]Note:[/b] [SceneTree]'s group methods will [i]not[/i] work on this node if not inside the tree (see [method is_inside_tree]).
 			</description>
 		</method>
+		<method name="atr" qualifiers="const">
+			<return type="String" />
+			<param index="0" name="message" type="String" />
+			<param index="1" name="context" type="StringName" default="&quot;&quot;" />
+			<description>
+				Translates a [param message], using the translation catalogs configured in the Project Settings. Further [param context] can be specified to help with the translation.
+				If [method Object.can_translate_messages] is [code]false[/code], or no translation is available, this method returns the [param message] without changes. See [method Object.set_message_translation].
+				This method works the same as [method Object.tr], with the addition of respecting the [member auto_translate_mode] state.
```

Comment:
This paragraph says the most important part. The fact that it automatically translates depending on the `auto_translate_mode` should appear much sooner in the description. Otherwise it's very easy to (accidentally) mix it around with `tr`.

(Same goes for `atr_n`)

Reply:
Done.